sdk/security_keyvault/src/keys/models.rs

```diff
@@ -406,10 +406,7 @@ pub struct EncryptResult {
 
 #[derive(Debug, Deserialize)]
 pub struct GetRandomBytesResult {
-    #[serde(
-        rename = "value",
-        serialize_with = "ser_base64",
-        deserialize_with = "deser_base64"
-    )]
+    /// `value` is encoded as a base64url string.
+    #[serde(rename = "value", deserialize_with = "deser_base64")]
```

Comment:
Ah, I see we're already doing that with this function. I forgot it was locally implemented.